cdap-docs/developers-manual/source/getting-started/building-apps.rst

```diff
@@ -93,44 +138,44 @@ Once an application is deployed:
     * - Parameter
       - Description
     * - ``<program-type>``
-      - One of ``flow``, ``mapreduce``, ``workflow`` or ``service``
+      - One of ``adapter``, ``flow``, ``mapreduce``, ``service``, ``spark``, ``worker``, or ``workflow``
     * - ``<app-id>``
-      - Name of the Application being called
+      - Name of the application being called
     * - ``<program-id>``
-      - Name of the *Flow*, *MapReduce*, *Workflow*, or *Custom Service*
-        being called
+      - Name of the *adapter*, *flow*, *MapReduce*, *service*, *spark*, *worker* or *workflow* being called
+      
 
 Stopping an Application
-----------------------------------
+-----------------------
 
 Once an application is deployed:
 
-- On the Application's detail page in the CDAP UI, you can click on a program to go 
+- On an application's detail page in the CDAP UI, you can click on a program to go 
   to its detail page and then click the *Stop* button there; or
-- From the Standalone CDAP SDK directory, use the :ref:`Command Line Interface: <cli>`
-
-  .. list-table::
-    :widths: 20 80
-    :stub-columns: 1
-
-    * - On Linux:
-      - ``$ ./bin/cdap-cli.sh stop <program-type> <app-id.program-id>``
-    * - On Windows:
-      - ``> bin\cdap-cli.bat stop <program-type> <app-id.program-id>``
+- From the Standalone CDAP SDK directory, use the :ref:`Command Line Interface <cli>`::
 
+    $ cdap-cli.sh stop <program-type> <app-id.program-id>
+    
+    
 Removing an Application
-----------------------------------
+-----------------------
 
-Once an application is stopped |---| all Processes (Flows, MapReduce programs, Workflows,
-etc.), Queries, and Services are stopped |---| you can go to the *Management* page of the
-CDAP UI, click on the Namespaces > default > Applications to bring up all Applications, 
-and then click on a particular Application, and then from the Application's detail page,
-select the *Actions > Delete* menu in the upper right to delete the Application. 
+Once an application is stopped |---| all of its programs (flows, MapReduce programs,
+workflows, etc.), queries, and services are stopped |---| you can go to the |all_apps|_
```

Comment:
I believe that queries and services belong in the parentheses, since that is a continuation of the list of programs.
Also, is 'queries' is no longer a program type, right?